genre.rb

```diff
@@ -0,0 +1,25 @@
+require_relative './item'
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name)
+    @name = name
+    @id = rand(1...1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item unless @items.include?(item)
+    item.genre = self
+  end
+end
+
+# genre =Genre.new('Way to the moon')
+
+# genre.add_item('hipop')
+# genre.add_item('hindi')
+# genre.add_item('work')
+# genre.add_item('Rumba')
+
+# print genre.items
```

Comment:
- [ ] [ OPTIONAL ] Once your project is ready for deployment, kindly remove the commented codes to make your code clean